python/cuda_parallel/cuda/parallel/experimental/_bindings.py

```diff
@@ -41,3 +43,22 @@ def get_paths() -> List[bytes]:
         if path is not None
     ]
     return paths
+
+
+def call_build(build_impl_fn: Callable, args: Tuple):
```

Comment:
Can we accept `*args`, `**kwargs` here?

Reply:
Absolutely, but for all cases of interest it would be empty. Let me add it.

Reply:
Done